portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile/MyTestFile.macro

```diff
@@ -1,13 +0,0 @@
-definition {
-    
-    // ValidateString
-    macro assertConfiguration {
-        AssertVisible(locator1 = "MyTestFile#CONFIGURATION");
-    }
-    macro clickControPanel {
-        Click(locator1 = "MyTestFile#BUTTON_CONTROL_PANEL");
-    }
-    macro clickMenu {
-        Click(locator1 = "MyTestFile#BUTTON_MENU");
-    }
-}
```

Comment:
Why you are adding deleted parts? Reset to the head of your branch, put all code in the right place and commit correctly, thanks.